src/js/tables.js

```diff
@@ -1,11 +1,14 @@
-/*global require*/
 /**
  * Wrap tables
  */
-'use strict';
 
-var $ = require('jquery');
+'use strict';
 
-$(function() {
-	$('.o-techdocs-content > table').wrap('<div class="o-techdocs-table-wrapper"></div>');
-});
+document.addEventListener('o.DOMContentLoaded', function() {
+	Array.from(document.querySelectorAll('.o-techdocs-content > table')).forEach(function(el) {
+		var wrapper = document.createElement('div');
+		wrapper.className = "o-techdocs-table-wrapper";
```

Comment:
We don't have a standard for this, so mixed quote style is fine.

I really don't get this.  I see no benefit to that consistency.  Does it seriously make the code clearer that the quote style is always the same?  What about strings that contain quoted strings?

```
var css = 'background: url("data:image/png...");';
```

You can't know that every type of content that might be in a string in JS source code is subject to complimentary quote style standards.  How about a quoted string inside a quoted string inside a third string?  There are only so many quote characters!  Escaping is OK, and frankly I think the intent here is clearer:

```
var css = "background: url(\"data:image/png...\");";
```

You now know at a glance that the inner quote chars are not 'active' because they are escaped.  Anyway, I'll leave the code as it is for the moment since there's no style rule for this in the spec.  If we really have to have another conversation about quote style I'm happy to just go with the majority view and write a standard for it if that's what people want, but I'd rather not.